### PR TITLE
VACMS-17057 Events detail page web component upgrades

### PR DESCRIPTION
**Pull request by randimays**
## Summary
Update individual event template with web components where they exist.

## Related issue(s)
- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17057

## Testing done
Tested a couple of events pages locally:
- `/minneapolis-health-care/events/64415/`
- `/outreach-and-events/events/53182/`

### Online event link
Screenshot 2024-03-18 at 2 14 19 PM
Screenshot 2024-03-18 at 2 14 13 PM

### See more events link
Screenshot 2024-03-18 at 2 14 05 PM
Screenshot 2024-03-18 at 2 13 58 PM

### Show all times button
Screenshot 2024-03-18 at 2 13 54 PM
Screenshot 2024-03-18 at 2 13 49 PM

### Add to calendar links inside accordion
Screenshot 2024-03-18 at 2 13 35 PM
Screenshot 2024-03-18 at 2 13 30 PM

### More times accordion
Screenshot 2024-03-18 at 2 13 23 PM
Screenshot 2024-03-18 at 2 13 10 PM

### Single add to calendar link
Screenshot 2024-03-18 at 2 12 40 PM
Screenshot 2024-03-18 at 2 12 32 PM

## Acceptance criteria

- [x] Links are using V1
- [x] Accordion is using V3
- [x] other features display correctly
- [x] verify/review the E2E tests
- [x] check desktop and mobile
- [x] a11y review

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] Linting warnings have been addressed
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### CODE REVIEW

**Review by eselkin — APPROVED** [review]
LGTM

**Comment by laflannery** [comment]
@randimays A few comments:
- I'm comparing an event on [the PR environment](http://eea85ecc9260cefdf0d667b0129d0f41.review.vetsgov-internal/outreach-and-events/events/64652/) to the same event on [Live](https://www.va.gov/outreach-and-events/events/64652/)
- The share links are using the `<va-link>` and are under the "Add to Calendar" Link but I can't see them visually, I can tab to them though.
- The "Last updated" text and the feedback button isn't on the page anymore. Was this intentional?
- Because there is only 1 accordion for the recurring events, as we able to remove The "Expand All" option from the component?

File: src/site/includes/social-share.drupal.liquid [hunk 1]
```diff
@@ -4,31 +4,35 @@
...
-        <i class="va-c-social-icon fas fa-calendar-check vads-u-margin-right--0p5" aria-hidden="true" role="presentation"></i>
-        Add to Calendar
-      </a>
+        text="Add to Calendar"
+      />
     </li>
 
     <li class="vads-u-margin-bottom--2p5">
-      <a class="va-js-share-link" href="https://www.facebook.com/sharer/sharer.php?href={{ hostUrl }}{{ entityUrl.path }}">
+      <va-link
```
Comment (chriskim2311) [inline src/site/includes/social-share.drupal.liquid:21]: These va-links are not showing up for events. I had to remove the icon and add text for the link to show up properly in clps
```
<i class="va-c-social-icon fab fa-facebook vads-u-margin-right--0p5" aria-hidden="true"></i>
     <va-link class="va-js-share-link" href="https://www.facebook.com/sharer/sharer.php?href={{ hostUrl }}{{ entityUrl.path }}" text="Share on Facebook">
     </va-link>
```

File: src/site/layouts/event.drupal.liquid [hunk 2]
```diff
@@ -219,68 +218,54 @@
...
+            </va-accordion-item>
+          </va-accordion>
         {% endif %}
 
         <!-- See more events. -->
-        <a
-          class="vads-u-padding-bottom--3 vads-u-margin-top--1 vads-u-font-weight--bold"
+        <va-link
+          active
+          class="vads-u-padding-bottom--3 vads-u-margin-top--1"
```
Comment (chriskim2311) [inline src/site/layouts/event.drupal.liquid:263]: For certain events the padding bottom is being overwritten. Not sure if we want to fix this or not. 
`/outreach-and-events/events/65872/`

Outreach_And_Events___Bourbiz_DC_At_The_MGM_National_Harbor___Veterans_Affairs_and_Downloads

**Review by chriskim2311 — APPROVED** [review]
Changes look good approved!

**Pull request merged**